docs/docs/manuals/error-messages.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Error Messages
```

Comment:
```suggestion
title: Troubleshoot
```

Reply:
I think we should call this troubleshoot, as it is the location for users to visit when something brakes.